### ISSUE TITLE
[Luis] Scene 5 - Job discrimination portion feels out of place

### ISSUE THREAD

**Issue by MicheleAWilliams-A11y** [w3c/wai-people-use-web-videos#133]
While it may be important to bring up greater issues like job discrimination, I'm not sure what we're supposed to get from introducing that then talking about online calculators and dictionaries, particularly since it doesn't say how that might relate to the jobs Luis is applying to. 

Also, job application forms are some of the worst websites to use currently so (if people are paying attention) it feels unlikely he would have put in several applications if a ticket website is hard to use. 

If we're trying to call out the need for simplified language here, I think there's other ways to do that. In the Luis persona doc, for instance, it says that this relates to general workplace jargon. That makes a little more sense.

**Comment by nitedog** [comment]
Changed entire scene to now say:

> I work part-time at a library, sorting books. I'd like to work full-time in an office job but I'm not sure I can do it. I have trouble with difficult words and with math. I use an online dictionary and calculator to help me but it takes time. Sometimes it is not just the words but the text is hard to follow. I get lost when the text is too long.

**Comment by MicheleAWilliams-A11y** [comment]
This is fine for this particular fix; I'm having an overall issue with this script that I'll put in other ticket.
